src/Project/Project.router.ts

```diff
@@ -79,6 +79,15 @@ export class ProjectRouter extends Router {
       server.handleRequest(this.deleteProject)
     )
 
+    /**
+     * Update all projects with coords to null
+     */
+    this.router.delete(
+      '/projects/coords/:coords',
```

Comment:
I think we could make this a bit more RESTful if we rename it to `/projects/:coords/coords` right? maybe we can think of an action as the last path instead of just coords, but at least the DELETE verb is semantic

Reply:
Sure! It has been changed!